src/sharedTypes.ts

```diff
@@ -33,3 +33,9 @@ export interface Word {
 }
 
 export type MediaFileExtension = AudioFileExtension | VideoFileExtension;
+
+export interface OperatingSystems {
```

Comment:
Same idea, if we do this on the back end then no shared types needed (also again, would make this an enum)

Reply:
enum refactor here: https://github.com/patrickbrett/mlvet/pull/38/commits/f54cbe3641987ce328de2f5e4091eb81a6eb50d4